### PR TITLE
Event list toggles

### PR DESCRIPTION
**Pull request by jmarx**
#124 Hey @mauteri I have a question on this requirement for the Featured Image toggle.
(include option to change image crop)

Do you want a dropdown menu with fixed crops( 4:3, 16:9, etc  )? Or simply the ability to change image sizes?
Could you please clarify the desired behavior?

### CODE REVIEW

File: assets/src/blocks/events-list/edit.js [hunk 1]
```diff
@@ -166,11 +170,84 @@ const Edit = ( props ) => {
 						maxSuggestions={ 20 }
 					/>
 				</PanelBody>
+				<PanelBody>
+					<ToggleControl
+						label="Show/Hide Attendee list"
```
Comment (mauteri) [inline assets/src/blocks/events-list/edit.js:175]: `label={ __( 'Show/Hide Attendee list', 'gatherpress' ) }`

File: assets/src/blocks/events-list/edit.js [hunk 2]
```diff
@@ -166,11 +170,84 @@ const Edit = ( props ) => {
...
+						help={
+							attributes.showAttendeeList
+								? 'Show Attendee List'
+								: 'Do not show Attendee List'
```
Comment (mauteri) [inline assets/src/blocks/events-list/edit.js:179]: Internationalize lines 178 and 179 (see example above)

File: assets/src/blocks/events-list/edit.js [hunk 3]
```diff
@@ -166,11 +170,84 @@ const Edit = ( props ) => {
...
+						}
+						checked={ attributes.showAttendeeList }
+						onChange={ () => {
+							setAttributes( { showAttendeeList: ! attributes.showAttendeeList } );
+						} }
+					/>
+					<SelectControl
+						label="Image Size Options"
```
Comment (mauteri) [inline assets/src/blocks/events-list/edit.js:187]: Internationalize

File: assets/src/blocks/events-list/edit.js [hunk 4]
```diff
@@ -166,11 +170,84 @@ const Edit = ( props ) => {
...
+						value={ attributes.imageSize }
+						options={ imageOptions }
+						onChange={ ( value ) => {
+							setAttributes( { imageSize: value } );
+						} }
+					/>
+					<ToggleControl
+						label="Show/Hide Featured Image"
```
Comment (mauteri) [inline assets/src/blocks/events-list/edit.js:195]: Internationalize

File: assets/src/blocks/events-list/edit.js [hunk 5]
```diff
@@ -166,11 +170,84 @@ const Edit = ( props ) => {
...
+						help={
+							attributes.showFeaturedImage
+								? 'Show Featured Image'
+								: 'Do not show Featured Image'
```
Comment (mauteri) [inline assets/src/blocks/events-list/edit.js:199]: Internationalize lines 198 and 199

File: assets/src/blocks/events-list/edit.js [hunk 6]
```diff
@@ -166,11 +170,84 @@ const Edit = ( props ) => {
...
+						onChange={ () => {
+							setAttributes( { showFeaturedImage: ! attributes.showFeaturedImage } );
+						} }
+					/>
+					<ToggleControl
+						label="Show/Description"
+						help={
+							attributes.showDescription
+								? 'Show Description'
+								: 'Hide Description'
```
Comment (mauteri) [inline assets/src/blocks/events-list/edit.js:211]: Internationalize lines 210 and 211

File: assets/src/blocks/events-list/edit.js [hunk 7]
```diff
@@ -166,11 +170,84 @@ const Edit = ( props ) => {
...
+						}
+						checked={ attributes.showDescription }
+						onChange={ () => {
+							setAttributes( { showDescription: ! attributes.showDescription } );
+						} }
+					/>
+					<TextControl
+						label="Description Limit"
+						help="Limit the amount of words that display underneath the title of the event"
```
Comment (mauteri) [inline assets/src/blocks/events-list/edit.js:220]: Internationalize lines 219 and 220

**Review by mauteri — CHANGES_REQUESTED** [review]
Looks pretty good! Just some small changes pertaining to internationalization.

File: assets/src/blocks/events-list/edit.js (hunk 1 above)
Comment (jmarx) [reply assets/src/blocks/events-list/edit.js]: @mauteri done.

File: assets/src/blocks/events-list/edit.js (hunk 2 above)
Comment (jmarx) [reply assets/src/blocks/events-list/edit.js]: @mauteri done.

File: assets/src/blocks/events-list/edit.js (hunk 3 above)
Comment (jmarx) [reply assets/src/blocks/events-list/edit.js]: @mauteri done.

File: assets/src/blocks/events-list/edit.js (hunk 4 above)
Comment (jmarx) [reply assets/src/blocks/events-list/edit.js]: @mauteri done.

File: assets/src/blocks/events-list/edit.js (hunk 5 above)
Comment (jmarx) [reply assets/src/blocks/events-list/edit.js]: @mauteri done.

File: assets/src/blocks/events-list/edit.js (hunk 6 above)
Comment (jmarx) [reply assets/src/blocks/events-list/edit.js]: @mauteri done.

File: assets/src/blocks/events-list/edit.js (hunk 7 above)
Comment (jmarx) [reply assets/src/blocks/events-list/edit.js]: @mauteri done.

**Review by mauteri — APPROVED** [review]
Looks good to me! Ready for me to merge?

**Comment by jmarx** [comment]
@mauteri Sure! You said at WordCamp you wanted to pull it down and run some tests ?

**Pull request merged**